pkgs/applications/networking/mailreaders/thunderbird/packages.nix

```diff
@@ -1,17 +1,15 @@
 { stdenv, lib, buildMozillaMach, callPackage, fetchurl, fetchpatch, nixosTests, icu73, fetchpatch2, config }:
 
-rec {
-  thunderbird = thunderbird-115;
-
-  thunderbird-115 = (buildMozillaMach rec {
+let
+  common = { version, sha512, updateScript }: (buildMozillaMach rec {
     pname = "thunderbird";
-    version = "115.12.2";
+    inherit version updateScript;
     application = "comm/mail";
     applicationName = "Mozilla Thunderbird";
     binaryName = pname;
     src = fetchurl {
       url = "mirror://mozilla/thunderbird/releases/${version}/source/thunderbird-${version}.source.tar.xz";
-      sha512 = "182f35e8e5ece98d18dfefe106c73bc97fbc619f59772d9b3455b7c8af412021ecc5eae97a12515224e91deb814abb7a6ef7f538c450e9e77fdfd84078678038";
+      inherit sha512;
     };
     extraPatches = [
       # The file to be patched is different from firefox's `no-buildconfig-ffx90.patch`.
```

Comment:
Does this still need to be patched? Filename references 115.

Reply:
I haven't checked (yet), but it feels natural that "build config" would contain various references that we don't want (saved build flags or something).

Reply:
So I tried.  The difference in closure size for the unwrapped thunderbird is 0.72G vs. 4.6G.  Let me drop the confusing number from the filename.

Reply:
It's rust, gcc, two clangs and more ... insane bloat.